docs/server/source/quickstart.md

```diff
@@ -6,6 +6,20 @@ A. Install MongoDB as the database backend. (There are other options but you can
 
 [Install MongoDB Server 3.4+](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/)
 
+Default database directory where mongodb stores its data: /data/db on Linux and OS X, \data\db on Windows.
+
+If you want to run mongodb with default database path i.e. /data/db, Open a Terminal and run the following commands as root/admin user:
+```text
+$ mkdir -p /data/db
+```
+
+Assign rwx(read/write/execute) permissions to the user for default database directory:
+```text
+$ sudo chmod -R 700 /data/db
+```
+
+**Note:** If you installed MongoDB using a package management system, check the /etc/mongod.conf file provided by your packages to see the directory is specified. If you are using a custom database path, the files in --dbpath must correspond to the storage engine specified in --storageEngine. If the data files do not correspond to --storageEngine, mongod will refuse to start.
```

Comment:
This Note feels like too much information for the Quickstart page. Let's cut it. If they run into problems, I think we'll have to let them either as us for help or Google it.